bin/recog_verify

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env ruby
 
 $:.unshift(File.expand_path(File.join(File.dirname(__FILE__), "..", "lib")))
+require 'nokogiri'
 require 'optparse'
 require 'ostruct'
 require 'recog'
-require 'recog/verifier_factory'
+require 'recog/formatter'
+require 'recog/verifier'
+require 'recog/verify_reporter'
 
-options = OpenStruct.new(color: false, detail: false, quiet: false, warnings: true)
+options = OpenStruct.new(color: false, detail: false, quiet: false, warnings: true, schema_file: nil)
```

Comment:
validation against the xsd is *not* on by default. You have to explicitly enable it. Figured this was the best option as to not require folks to now know where the xsd file is every time they want to use recog_verify.